packages/mdc-textfield/foundation.js

```diff
@@ -150,4 +156,20 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
       disabled: false,
     };
   }
+
+  registerAutoCompleteHandler_(input) {
+    return new Promise((resolve, reject) => {
+      let hasKeyInteraction = false;
+
+      input.addEventListener('input', (e) => {
+        if (!hasKeyInteraction) {
+          resolve(e);
```

Comment:
do we need to call resolve with the event object? It doesn't look like we use it.